generator/src/swam/generator/Generator.scala

```diff
@@ -0,0 +1,90 @@
+package swam.generator
+
+import java.io.File
+import java.util.concurrent.Executors
+
+import cats.effect.{Blocker, IO}
+import org.json4s.DefaultFormats
+import swam.runtime.Engine
+import org.json4s.jackson.Serialization.writePretty
+
+import scala.concurrent.ExecutionContext
+case class Config(wasms: Seq[File] = Seq(),
+                  printTemplateContext: Boolean = false,
+                  createBoilerplate: String = "",
+                  renderTemplate: File = null)
+
+/**
+    @author Javier Cabrera-Arteaga on 2020-03-07
+  */
+object Generator extends App {
+
+  implicit val cs = IO.contextShift(ExecutionContext.Implicits.global)
+
+  val blockingPool = ExecutionContext.fromExecutorService(Executors.newCachedThreadPool())
+  val blocker: Blocker = Blocker.liftExecutionContext(blockingPool)
```

Comment:
You should use the [`Resource`](https://typelevel.org/cats-effect/datatypes/resource.html) based `Blocker` as demonstrated [here](https://typelevel.org/cats-effect/datatypes/contextshift.html#blocker).

Reply:
I think is fixed. But, honestly, I think I am not understanding cats very wel

Reply:
The idea of cats-effect is that you can write code with side-effects in a pure way. Since this is intended to be integrated into swam, I'd like to have the code up to the standards of the rest of the code. If you don't feel like you get it enough, I can take your changes and rewrite the parts that need to be improved.